src/controller/point-of-sale-controller.ts

```diff
@@ -361,6 +368,49 @@ export default class PointOfSaleController extends BaseController {
     }
   }
 
+  /**
+   * GET /pointsofsale/{id}/associates
+   * @summary Returns a Point of Sale's associate users
+   * @operationId getPointOfSaleAssociates
+   * @tags pointofsale - Operations of the point of sale controller
+   * @param {integer} id.path.required - The id of the Point of Sale of which to get the associate users.
+   * @security JWT
+   * @return {PointOfSaleAssociateUsersResponse} 200 - The requested Point of Sale transactions
+   * @return {string} 404 - Not found error
+   * @return {string} 500 - Internal server error
+   */
+  public async returnPointOfSaleAssociates(req: RequestWithToken, res: Response): Promise<void> {
+    const { id } = req.params;
+    const pointOfSaleId = parseInt(id, 10);
+    this.logger.trace('Get Point of Sale associate users', id, 'by user', req.token.user);
+
+    try {
+      // Point of sale does not exist.
+      const pos = await PointOfSale.findOne({
+        where: { id: pointOfSaleId },
+        relations: { cashierRoles: true, owner: true },
+      });
+      if (!pos) {
+        res.status(404).json('Point of Sale not found.');
+        return;
+      }
+
+      const { owner } = pos;
+      const ownerMembers = await UserService.getUsers({ organId: pos.owner.id });
```

Comment:
I still think that a 'User.find(options)` would be better, since you can ommit the whole .records, but I don't think this has to change.